doc/source/building/working_with_images/iso_to_usb_stick_file_based_deployment.rst

```diff
@@ -0,0 +1,92 @@
+
+In KIWI, all generated ISO images are created to be hybrid. This means,
+the image can be used as a CD/DVD or as a disk. The deployment of such
+an image onto a disk like an USB stick normally destroys all existing
+data on this device. Most USB sticks are pre-formatted with a FAT32
+Windows File System and to keep the existing data on the stick untouched
+a different deployment needs to be used.
+
+The following deployment process copies the ISO image as an
+additional file to the USB stick and makes the USB stick bootable.
+The ability to boot from the stick is configured through a SYSLINUX
+feature which allows to loopback mount an ISO file and boot the
+kernel and initrd directly from the ISO file.
+
+The initrd loaded in this process must also be able to loopback
+mount the ISO file to access the root filesystem and boot the
+live system. The dracut initrd system used by KIWI provides this
+feature upstream called as "iso-scan". Therefore all KIWI generated
+live ISO images supports this deployment mode.
+
+For copying the ISO file on the USB stick and the setup of the
+SYSLINUX bootloader to make use of the "iso-scan" feature an extra tool
+named `live-fat-stick` exists. The following procedure shows how
+to setup the USB stick with `live-fat-stick`:
+
+1. Install the `live-fat-stick` package:
+
+   The package is available for openSUSE at:
+   https://software.opensuse.org/package/live-fat-stick
+
+2. Plug in a USB stick
+
+   Once plugged in, check which Unix device name the FAT32 partition
+   was assigned to. The following command provides an overview about all
+   storage devices and their filesystems:
+
+   .. code:: bash
+
+      $ sudo lsblk --fs
+
+3. Call the `live-fat-stick` command as follows:
+
+   Assuming "/dev/sdz1" was the FAT32 partition selected from the
+   output of the `lsblk` command:
+
+   .. code:: bash
+
+      $ sudo live-fat-stick LimeJeOS-Leap-42.3.x86_64-1.42.3.iso /dev/sdz1
+
+4. Update the SYSLINUX configuration on the USB Stick
+
+   By default, the SYSLINUX configuration does not use the "iso-scan"
+   feature. Thus this information needs to be added as follows:
+
+   * Mount the FAT32 partition of the USB Stick. This must be the
+     same device as used in the `live-fat-stick` command call:
+
+     .. code:: bash
+
+         $ sudo mount /dev/sdz1 /mnt
+
+   * Edit the :file:`/mnt/boot/syslinux/syslinux.cfg` and include the
+     following parameters to the existing **append** line:
+
+     .. code:: bash
+
+         iso-scan/filename=/LimeJeOS-Leap-42.3.x86_64-1.42.3.iso root=live:CDLABEL=CDROM
+
+   * Unmount the FAT32 partition:
+
+     .. code:: bash
+
+         $ sudo umount /mnt
+
+5. Boot from your USB Stick
+
+   Activate booting from USB in your BIOS/UEFI. As many firmware has different
+   procedures on how to do it, look into your user manual.
+   Many firmware offers a boot menu which can be activated at boot time.
+   Usually this can be reached by :kbd:`Esc` or :kbd:`F12`.
```

Comment:
I forget one thing in my last suggestion, sorry. Maybe use the following phrase:

```
Usually this can be reached by pressing the :kbd:`Esc` or :kbd:`F12` keys.
```

Reply:
I really hope nobody who does not know that ESC or F12 is a key on the keyboard and needs to be pressed to use it, will read this document :) :)

Reply:
Hehe, yes. Maybe I was a bit too exaggerating. 😉 Yes, maybe in this context the first version is enough.

Just to give you a little background why I did that.

From a writing perspective, I think it can be useful to have "the thing" named correctly. For example, sometimes it could be misleading if you don't have markup or font changes. A text like "Use File" without any style changes could mean anything (a menu item, a link, or something else). When you use "Use the File menu to..." the reader gets a better perspective what to look for.